content/docs/user-guide/networking/multiplayer/configuration.md

```diff
@@ -5,7 +5,65 @@ description: Learn how to add multiplayer support to an Open 3D Engine (O3DE) pr
+
+## Creating a multiplayer project from the multiplayer template {#creating-project-from-template}
+
+To create a multiplayer project from the O3DE multiplayer template, you will need O3DE version 22.10 or later. You must first add the template from the [O3DE Extras GitHub repo](https://github.com/o3de/o3de-extras) to your library of O3DE templates using **Project Manager**. Then you will be able to create a new project from the template.
+
+1. In Project Manager, choose **New Project**, then **Create New Project**.
+
+1. Choose **Add remote template**.
+
+1. In the **Add a remote template** dialog box, enter `https://github.com/o3de/o3de-extras.git` as the Remote URL. Then choose **Add**. This registers the GitHub repo as a remote source.
+
+1. Select the new template named **Multiplayer**. This selects the template for the new project.
+
+1. Choose **Download Template** (in the bottom right of the New Project window), then choose the location where the template will be downloaded. Choose **Download**.
+
+1. With the template downloaded, you can configure your project with additional Gems. Choose **Create Project** when ready to create the new project.
+
+1. Choose **Build Project - Build Now** from the project's icon box to build the project.
+
+1. When the build completes, your project will be ready to use. You can choose **Open Editor** to open it in **O3DE Editor**.
+
+1. The project comes with a `Demo` level which you can open from the Editor and play with using **Ctrl+G**.
+
+### Alternative ways to set up the multiplayer template
+
+To set up the multiplayer template without using Project Manager, you will need to download the template from the [O3DE Extras GitHub repo](https://github.com/o3de/o3de-extras) and then register the template from a command line.
+
+1. When obtaining the template from the O3DE Extras GitHub repo, you can choose to download a tagged release version or use `git clone` to get the latest prerelease code from the `development` branch. Do _one_ of the following:
+
+    a. To download a release version from the repo, open the [O3DE Extras tags page](https://github.com/o3de/o3de-extras/tags). Select a version, then choose the zip file that begins with the name `Template_Multiplayer` to download. Unzip the contents to a directory of your choice.
+
+    b. To download the latest prerelease code from the `development` branch, open a command line window on your computer and use `git clone` to get the contents of the repo:
+
+    ```cmd
+    git clone https://github.com/o3de/o3de-extras.git
+    ```
+
+1. Open a command line window to your O3DE engine directory and use the `o3de register` command to register the multiplayer template.
+
+    ```cmd
+    // Example using downloaded and unzipped release version.
+    scripts\o3de.bat register -tp C:/downloads/template_multiplayer
```

Comment:
We may want to specify the download folder using the default for web browsers.
`C:/Users/<username>/Downloads`
```suggestion
    // Example using downloaded and unzipped release version.
    scripts\o3de.bat register -tp C:/Users/<YourUserName>/Downloads/template_multiplayer
```